### DISCUSSION TITLE
Kustomize remote base?

### DISCUSSION THREAD

**Discussion by asgeirn** [argoproj/argo-workflows#7484] [Q&A] [answered]
According to [manifests/README](https://github.com/argoproj/argo-workflows/blob/cda7cd46a7ba6632cf6cb9ce034dabd48a491595/manifests/README.md?plain=1#L20), [y]ou can use `./cluster-install` and `./namespace-install` as Kustomize remote bases.

However, if you add `?ref=v3.2.6`, the image tags are still `latest`:
- https://github.com/argoproj/argo-workflows/blob/db7d90a1f609685cfda73644155854b06fa5d28b/manifests/base/argo-server/argo-server-deployment.yaml#L17
- https://github.com/argoproj/argo-workflows/blob/db7d90a1f609685cfda73644155854b06fa5d28b/manifests/base/workflow-controller/workflow-controller-deployment.yaml#L17

Is this variant actually recommended instead of using the manifest files from a release?

**Comment by terrytangyuan** [comment, marked as answer]
I don't think that README recommends anything. If you want to use a specific image tag, then you'll want to use the manifests from a release, e.g. https://github.com/argoproj/argo-workflows/releases/download/v3.2.6/install.yaml. If you use anything in `argo-workflows/manifests`, you'll have to override the image version yourself.

**Reply by asgeirn** [reply]
Maybe not recommends as such, but perhaps not clarifying either? In the documentation at https://argoproj.github.io/argo-workflows/installation/ using manifests attached to the tagged release is recommended.  Perhaps this README should direct users there or repeat these recommendations?

**Reply by terrytangyuan** [reply]
That's a good point. Would you like to submit a PR to improve related documentation?
